localstack/services/cloudwatch/alarm_scheduler.py

```diff
@@ -279,6 +285,10 @@ def calculate_alarm_state(alarm_arn: str) -> None:
     :param alarm_arn: the arn of the alarm to be evaluated
     """
     alarm_details = get_metric_alarm_details_for_alarm_arn(alarm_arn)
+    if not alarm_details:
```

Comment:
Nitty comment. This could be shortened with the walrus operator, i.e.,
```python
if not (alarm_details:= get_metric_alarm_details_for_alarm_arn(alarm_arn)):
    ... # log and return
```
Feel free to ignore the comment if you find it less readable.

Reply:
IMO it's more readable the way it is :) I prefer the walrus operator for any action that will be executed only if the parameter is set, but not the other way :)